### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by nikitashrestha**
In this PR, I have configured travis file and also created app folder that contains source code for our app.

[minor-changes: update travis for app and api
](https://github.com/leapfrogtechnology/resume-builder/commit/88425225ed32af417444f77314fb41891554de4e)
[minor-changes: create app folder to locate app source-codes
](https://github.com/leapfrogtechnology/resume-builder/commit/6e9d52bb1b5c6348a86977dee78be8803bc09ca7)
[minor-changes: Migrate CICD to travis ci
](https://github.com/leapfrogtechnology/resume-builder/commit/2a2835c0c79ce18672c4636b60b2670fdeed5a88)
[minor-changes: add TODO for deploy-bucket
](https://github.com/leapfrogtechnology/resume-builder/commit/742ac3d43a4643bffc3598274c8a81ee9ff319f0)
[update .env.example
](https://github.com/leapfrogtechnology/resume-builder/commit/4d3bff711eb3307546f6d72bf08adbd14fd63860)
[minor-changes b5a7beb] minor-changes: Add envault.json
[minor-changes 15d1a56] minor-changes: Add hosted zone
[minor-changes 780f1e3] minor-changes: Update profile

### CODE REVIEW

File: app/.env.example [hunk 1]
```diff
@@ -0,0 +1 @@
+GOOGLE_CLIENT_ID=''
```
Comment (leapfrog-saugat) [inline app/.env.example:1]: shouldn't this file have other keys as well?

Comment (nikitashrestha) [reply app/.env.example:1]: In app, we only require this key

Comment (nikitashrestha) [reply app/.env.example:1]: What other keys are we expecting here?

Comment (leapfrog-saugat) [reply app/.env.example:1]: ok. I guess we will be adding sentry pretty soon.

**Review by leapfrog-saugat — APPROVED** [review]

**Pull request merged**